ores/logging/logstash_fomatter.py

```diff
@@ -0,0 +1,85 @@
+import traceback
+import logging
+import socket
+from datetime import datetime
+import json
+
+
+class LogstashFormatter(logging.Formatter):
+    # The list contains all the attributes listed in
+    # http://docs.python.org/library/logging.html#logrecord-attributes
+    skip_list = (
+        'args', 'asctime', 'created', 'exc_info', 'filename', 'funcName',
+        'levelname', 'levelno', 'lineno', 'message', 'module', 'msecs',
+        'msg', 'name', 'pathname', 'process', 'processName',
+        'relativeCreated', 'stack_info', 'thread', 'threadName')
+    easy_types = (str, bool, dict, float, int, list, type(None))
+
+    def __init__(self, message_type='Logstash', tags=None, host=None):
+        self.message_type = message_type
+        self.tags = tags if tags is not None else []
+        self.host = host if host is not None else socket.gethostname()
+
+    def format(self, record):
+        # Create message dict
+        message = {
+            '@timestamp': self.format_timestamp(record.created),
+            '@version': '1',
+            'message': record.getMessage(),
+            'host': self.host,
+            'path': record.pathname,
+            'tags': self.tags,
+            'type': self.message_type,
```

Comment:
I am a bit confused about message_type. Will it always be "Logstash" ? and still sent over the wire ?

Reply:
Btw, https://wikitech.wikimedia.org/wiki/Logstash#Gotchas

```
Make sure logging events sent to the GELF input don't have a "type" or "_type" field set, or if set, that it contains the value "gelf". The gelf/logstash config discards any events that have a different value set for "type" or "_type". The final "type" seen in Kibana/Elasticsearch will be take from the "facility" element of the original GELF packet. The application sending the log data to Logstash should set "facility" to a reasonably unique value that identifies your application.
```

Reply:
Completely removed it.